bin/polysh

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
We should get rid of this in favour of `entry_points`/`console_scripts`.  See `setup.py` of one of our project for an example.

Reply:
Done